### DISCUSSION TITLE
Sandboxie SBIE2203 Error with Firefox 106.0+

### DISCUSSION THREAD

**Discussion by Michael-Germany** [sandboxie-plus/Sandboxie#2362] [Q&A Feedback] [answered]
Hello,

since the last Firefox update, an error message comes up from time to time. 
Opera runs slowly, could be the proxy setting, but for the short time without error.
So Firefox and Sandboxie have the problem.

My System: 
Windows 10 Pro 64Bit
Firefox 106.0
Sandboxie 5.58.2

**Sandboxie Error**
SBIE2203 Fehler bei der Kommunikation mit dem Sandboxie-Dienst: (Error communicating with the Sandboxie service:) *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]

**Help Messages form sandboxie-plus.com**

SBIE2203
Message: SBIE2203 Failed to communicate with Sandboxie Service: detail
Logged To: [Popup Message Log](https://sandboxie-plus.com/sandboxie/SBIE2203/PopupMessageLog).
Explanation:
The Sandboxie DLL component executing within the sandboxed program needs to communicate with the Sandboxie service component (SbieSvc). This message indicates that some communication failure has occurred.
When detail is connect, the likely reason is that the Sandboxie service is not running.
Any other value of detail indicates that communication has been established, but could not be completed, due to some error.

Please help
Michael-Germany

**Comment by Michael-Germany** [comment]
this update has helped
https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.5.0

**Comment by telefunker** [comment]
Hi everyone.
With the new Firefox Version 107.0 the problem occurs again

Edition	Windows 10 Pro
Version	21H2
Installiert am	‎26.‎04.‎2021
Betriebssystembuild	19044.2251
Leistung	Windows Feature Experience Pack 120.2212.4180.0

Any idea?

**Comment by telefunker** [comment, marked as answer]
Solved with Sandboxie 1.5.3 
https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.5.3
